db/seeds.rb

```diff
@@ -0,0 +1,45 @@
+require 'http'
+
```

Comment:
It also would be nice to see a separation of concerns, where the concerns of `seeds.rb` at the moment is twofold: fetching the data, parsing it by row/entry in the sheet, and persisting those records to the database. The last two can probably be combined into a single concern, but the former (fetching) should standalone.

- `lib/<app-name>/actions/fetch_bank_data.rb`
- `lib/<app-name>/actions/parse_bank_data.rb`

You can name them whatever you want though just providing an example.